action.yml

```diff
@@ -1,4 +1,4 @@
-name: Oppiabot Actions
+name: PraneethsOppiabot
```

Comment:
TODO: Revert after local testing is no longer needed.

Reply:
Can we remove it or is there some testing planned?

Reply:
A final test after @jameesjohn reviews it.

Reply:
This can be removed now.